Examples/ObjectiveC/RasterImageryExample.m

```diff
@@ -47,12 +48,25 @@ - (void)updateLayerOpacity:(UISlider *)sender {
 - (void)addSlider {
     CGFloat padding = 10.0;
     UISlider *slider = [[UISlider alloc] initWithFrame:CGRectMake(padding, self.view.frame.size.height - 44 - 30, self.view.frame.size.width - padding * 2, 44)];
-    slider.autoresizingMask = UIViewAutoresizingFlexibleTopMargin | UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin;
     slider.minimumValue = 0;
     slider.maximumValue = 1;
     slider.value = 1;
     [slider addTarget:self action:@selector(updateLayerOpacity:) forControlEvents:UIControlEventValueChanged];
     [self.view addSubview:slider];
+    
+    if (@available(iOS 11, *)) {
+        UILayoutGuide *safeArea = self.view.safeAreaLayoutGuide;
+        slider.translatesAutoresizingMaskIntoConstraints = NO;
+        NSArray *constraints = @[
+          [slider.bottomAnchor constraintEqualToAnchor: safeArea.bottomAnchor constant:-self.mapView.logoView.bounds.size.height],
```

Comment:
I'd consider subtracting a little bit of value from `self.mapView.logoView.bounds.size.height`, since the slider button overlaps the information icon a bit still:

<img width="428" alt="screenshot 2019-03-01 15 11 29" src="https://user-images.githubusercontent.com/10850812/53672853-34507000-3c39-11e9-9285-6e96aa5487d7.png">